code/game/objects/random/random.dm

```diff
@@ -442,7 +446,11 @@
 		/obj/item/clothing/suit/space/void/hos = 0.3,
 		/obj/item/clothing/suit/space/void/lancer = 0.3,
 		/obj/item/clothing/suit/space/void/sci = 2,
-		/obj/item/clothing/suit/space/void/sol = 0.5
+		/obj/item/clothing/suit/space/void/sol = 0.5,
+		/obj/item/clothing/suit/space/void/necropolis = 1,
+		/obj/item/clothing/suit/space/void/einstein = 1,
+		/obj/item/clothing/suit/space/void/hephaestus = 1,
+		/obj/item/clothing/suit/space/void/zenghu = 1
```

Comment:
These probabilities seem relatively high compared to the other items in the list.